client/src/app/menu/menu.component.ts

```diff
@@ -63,9 +70,31 @@ export class MenuComponent implements OnInit {
       }
     )
 
-    this.hotkeysService.cheatSheetToggle.subscribe(isOpen => {
-      this.helpVisible = isOpen
-    })
+    this.hotkeysService.cheatSheetToggle.subscribe(isOpen => this.helpVisible = isOpen)
+
+    this.serverService.getVideoLanguages().subscribe(languages => this.languages = languages)
+  }
+
+  get language () {
+    return this.languageChooserModal.getCurrentLanguage()
+  }
+
+  get videoLanguages (): string[] {
+    if (!this.user.videoLanguages) return [this.i18n('any language')]
```

Comment:
`this.user` could be undefined (even if you check user in the template I think it's a good behaviour to add a small check: `if (!this.user) return`)